functions/src/services/wetherspoons.service.test.js

```diff
@@ -0,0 +1,152 @@
+const { processDrinksResponse, getDrink } = require('./wetherspoons.service');
+
+// const expectedDrinkGenerator = (drinkName, numberOfDrinks) => {
+//   const drinks = [];
+//   for (let i = 0; i < numberOfDrinks; i++) {
+//     const drink = { displayName: `${drinkName}: ${i}` };
+//     drinks.push(drink);
+//   }
+//   return drinks;
+// }
```

Comment:
I think Im okay with this being commented out if not needed now but will be soon
But Bob Martin says that if you want to comment code out it should just be removed
See point 5: https://gist.github.com/wojteklu/73c6914cc446146b8b533c0988cf8d29#comments-rules